samples/word/Basics/custom-properties-access.yaml

```diff
@@ -0,0 +1,83 @@
+id: ciyyoh3d00000456y37cgjdki
+gist: ''
+source: WORD
+author: Microsoft
+name: GetCustomDocProperties
+description: Colors the selected text in red.
+script:
+  content: |-
+    $("#run").click(getProperties);
+
+    function getProperties() { 
+        Word.run(function (context) {
+            var customDocProps = context.document.properties.customProperties;
+            context.load(customDocProps);
+            return context.sync()
+                .then(function () {
+                    console.log(customDocProps.items.length);
+                 })
+         })
+
+
+    }
+  language: typescript
+style:
+  content: |-
+    body {
+        margin: 0;
+        padding: 10px;
+    }
+
+
+    /* Button customization, including overwriting some Fabric defaults */
+
+    .ms-Button, .ms-Button:focus {
+        background: #2b579a;
+        border: #2b579a;
+    }
+
+        .ms-Button > .ms-Button-label,
+        .ms-Button:focus > .ms-Button-label,
+        .ms-Button:hover > .ms-Button-label {
+            color: white;
+        }
+
+        .ms-Button:hover, .ms-Button:active {
+            background: #204072;
+        }
+
+        .ms-Button.is-disabled, .ms-Button:disabled {
+            background-color: #f4f4f4;
+            border-color: #f4f4f4;
+        }
+
+        .ms-Button.is-disabled .ms-Button-label,
+        .ms-Button:disabled .ms-Button-label {
+            color: #a6a6a6;
+        }
+  language: css
+template:
+  content: |-
+    <p class="ms-font-m">Executes a simple code snippet</p>
```

Comment:
Please see samples/excel/range/set-get-values.yaml for the format of the HTML. There should be a introduction, a setup button (if needed), and a set up buttons for each function to try. In this snippet there will only be one. The name of the button should be the function that the snippet will demonstrate.